test/tactics.lean

```diff
@@ -5,7 +5,7 @@ Authors: Simon Hudon, Scott Morrison
 -/
 
 import tactic.interactive tactic.finish tactic.ext tactic.lift tactic.apply
-       tactic.reassoc_axiom tactic.tfae tactic.elide tactic.ring_exp
+import tactic.reassoc_axiom tactic.tfae tactic.elide tactic.ring_exp
```

Comment:
Not done here?

Reply:
Ah, I only ran some regexes in the `src/` directory.

Reply:
Fixed.